app/cronjobs/recruiterreminder.php

```diff
@@ -0,0 +1,151 @@
+    $jobidResult = $db->applications->distinct("jobid",array("status"=>0));
+
+    //get unique recruiterids with unclaimed applications
+    $recruiteridResult = $db->jobs->distinct("recruiter", array('$or' => wrapIdsInArrayOfMongoIds($jobidResult)));
+
+    // query the recruiters collection for any recruiters with ids in $recruiteridResult, and also recruiters that have not been sent an email in a week or whose last_emailed field is NULL
+    $currentTime = time();
+    $interval = time() - 7 * 24 * 60 * 60; // 7 days before
+    $mongoInterval = new MongoDate($interval); // MongoDate seven days in the past
+
+    //final query
+    $recruiterResult = $db->recruiters->find(
+      array(
+          '$and' => array(
+              array(
+                '$or' => array(
+                  array(
+                    "last_emailed" => array(
+                      '$lt' => $mongoInterval
+                    )
+                  ),
+                  array(
+                    "last_emailed" => null
+                  )
+                )
+              ),
+              array(
+                '$or' => wrapIdsInArrayOfMongoIds($recruiteridResult)
+              )
+            )
+        )
+      )->limit(MAX_EMAILS);
+
+    if ($recruiterResult) {
+      $recruitersEmailed = []; // keep track of emails of recruiters reminded using this script
+      //set up phpmailer
+      $mail = new PHPMailer;
+      $mail->isSMTP();                                      // Set mailer to use SMTP
+      $mail->Host = 'smtp.gmail.com';                       // Specify main and backup server
+      $mail->SMTPAuth = true;                               // Enable SMTP authentication
+      $mail->Username = 'info@sublite.net';                 // SMTP username
+      $mail->Password = $gmailpass;
+      $mail->SMTPSecure = 'tls';                            // Enable encryption, 'ssl' also accepted
+      $mail->Port = 587;                                    //Set the SMTP port number - 587 for authenticated TLS
+      $mail->setFrom("info@sublite.net");
+      $mail->addReplyTo("info@sublite.net");
+      $mail->WordWrap = 80;
+      $mail->isHTML(true);
+      $subject = "SubLite Unclaimed Applications";
+      $mail->Subject = $subject;
+
+      foreach ($recruiterResult as $recruiter) { // loop through all recruiters and send them personalized reminder emails
+        $recruitersEmailed[] = $recruiter['email'];
+        $mail->addAddress($recruiter['email']);
+        $recruiterFullName = $recruiter['firstname'] . $recruiter['lastname'];
+        $recruiterId = $recruiter['_id'];
+        $linkDashboard = "http://sublite.net/employers/home"; //how to make a link to dashboard?
+
+        $message = "
+          Dear $recruiterFullName,
+          <br />
+          <br />You have unclaimed applications for some of your job listings! Please visit your <a href = '$linkDashboard'>recruiter homepage</a> and your invidual job listings pages to claim them!
```

Comment:
We should elaborate on this a bit and tell them about for example, how our credit system works, and tell them about our payment plans. We should also provide contact information. Talk to Dean or I about it